### PR TITLE
fix: branding for search page buttons

### PR DESCRIPTION
**Pull request by Enubia**
added cdn links back as comments so that we can revert the static
serving in case this doesn't work

### CODE REVIEW

**Comment by gnorbsl** [comment]
this might clash with my branch where I updated the scripts and added the map files, can we merge mine first and see if there are conflicts?

**Comment by Enubia** [comment]
> this might clash with my branch where I updated the scripts and added the map files, can we merge mine first and see if there are conflicts?

sure

**Comment by Enubia** [comment]
@gnorbsl ready for review

File: client/views/wishCards.ejs [hunk 1]
```diff
@@ -167,15 +173,14 @@
...
-                        data-target="#wishCardDonateModal"
-                      <% } else { %>
-                        data-target="#loginModalCenter"
-                              <% } %>
-                        class="wishcard__button--blue bdr-2"
+                        <%
+                        if
+                        (locals.user)
+                        {
+                        %
```
Comment (gnorbsl) [inline client/views/wishCards.ejs:180]: your autoformat really fucks stuff up in the ejs files

Comment (gnorbsl) [reply client/views/wishCards.ejs:180]: aside fromt that this reverts whats in master right now

**Review by gnorbsl — CHANGES_REQUESTED** [review]

**Review by gnorbsl — APPROVED** [review]

**Pull request merged**